data.json

```diff
@@ -143,6 +143,15 @@
     "dateAdded": "2018-03-11T16:47:14.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "NativeScript",
+    "difficulty": "easy",
+    "description": "Make your first contribution to the NativeScript community and win an awesome set of stickers. Among the stickers you will find a limited edition 'Contributor' sticker. ",
+    "reference": "https://www.nativescript.org/blog/nativescript-first-time-contribution-contest-continued-and-extended",
+    "image": "https://d2odgkulk9w7if.cloudfront.net/images/default-source/default-album/stickers-photo.jpg",
+    "dateAdded": "2019-01-25T08:36:39.000Z",
+    "tags": ["stickers", "surprise"]
```

Comment:
I'm not sure about the `surprise` tag. It's related to the limited edition Contributor sticker I guess?

Reply:
We will send extra special swag in the package for extra useful contributions. It's going to be a surprise when you receive it - hence the name of the tag. Makes sense?

Reply:
I definitely see where you're coming from @etabakov! However, I feel like this could hurt the UX, since this is the only opportunity that would be tagged as such, and there's implicitly more of an expectation with such a name.

Reply:
OK, I don't insist too much on this tag. I will remove it for now.